lib/ConsortiumFieldInventory/ConsortiumFieldInventory.js

```diff
@@ -0,0 +1,132 @@
+import get from 'lodash/get';
```

Comment:
```suggestion
import {
get,
identity,
noop,
} from 'lodash';
```

Reply:
Actually, the direct import of functions was made specifically so that the bundler would only take those functions used in the code, instead of the whole library.

- https://www.blazemeter.com/blog/import-lodash-libraries
- https://dev.to/yutro/lodash-import-done-right-290n
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -